client/chain_height.go

```diff
@@ -0,0 +1,112 @@
+package client
```

Comment:
Any reason why this client package was moved out from being inside the tests package like it is in Umee? Are we gonna eventually be using this package outside of there?

Reply:
I think we could eventually use it in other places. We have similar code in the price-feeder, relayer, etc and this code is not specific to anything related to the e2e tests - just connecting to the Ojo grpc endpoint and sending queries/transactions.